### PR TITLE
locales update

### PR DESCRIPTION
**Pull request by natiloomio**

### CODE REVIEW

File: config/locales/client.cs.yml [hunk 1]
```diff
@@ -1,546 +0,0 @@
-cs:
```
Comment (gdpelican) [inline config/locales/client.cs.yml]: Er, why are we removing the czech client translations?

**Comment by gdpelican** [comment]
Can't merge this yet as the build's erroring because client.cs.yml is missing; any idea why that is @natiloomio @robguthrie ?

**Comment by natiloomio** [comment]
No idea!

--

**Comment by gdpelican** [comment]
Okay, well I'm going to close this and hope the next one goes better 🙈

**Pull request closed without merging**